js/app.js

```diff
@@ -122,52 +63,57 @@ window.addEventListener('DOMContentLoaded', function() {
     showElement(messageArea, false);
   }
 
+  function displayResult(result) {
+    console.log('Sync successfully finished!', result);
+    var nbDeleted = 0;
+    var nbUpdated = 0;
+    var nbAdded = 0;
+    for (var op of result) {
+      switch (op.action) {
+        case 'created':
+          nbAdded++;
+          break;
+        case 'updated':
+          nbUpdated++;
+          break;
+        case 'deleted':
+          nbDeleted++;
+          break;
+      }
+    }
+    var message =
+    showMessage(
+      '<h2>Sync successfully finished:</h2>'+
+      'Summary:<br>' +
+      `<ul>
+        <li>${nbAdded} contact(s) added</li>
+        <li>${nbUpdated} contact(s) updated</li>
+        <li>${nbDeleted} contact(s) deleted</li>
+      </ul>`
+    );
```

Comment:
(I know it was already like this before, but I didn’t get to review it back then ;-))